sycl/include/sycl/ext/oneapi/experimental/device_architecture.hpp

```diff
@@ -13,6 +13,17 @@ inline namespace _V1 {
 namespace ext::oneapi::experimental {
 
 enum class architecture {
+  // If new element is added to this enum:
+  //
+  // Update
+  //   - sycl_ext_oneapi_device_architecture specification doc
+  //   - "-fsycl-targets" description in sycl/doc/UsersManual.md
+  //
+  // Add
+  //   - __SYCL_TARGET_<ARCH>__ to the compiler driver and to all places below.
+  //   - the unique ID of the new architecture in SYCL RT source code to support
+  //     querying the device architecture
```

Comment:
Nit: either all the bullets end with `.` (like the third one), or none of them do.